odoo/addons/base/models/res_users.py

```diff
@@ -697,6 +701,57 @@ def _has_group(self, group_ext_id):
     # for a few places explicitly clearing the has_group cache
     has_group.clear_cache = _has_group.clear_cache
 
+    def _get_access_count(self):
```

Comment:
Guidelines: compute method should be prefixed by _compute. You should also add the triggers for the computation.